internal/controllers/reconciliation/integration_test.go

```diff
@@ -472,6 +474,7 @@ func TestReconcileStatus(t *testing.T) {
 	comp.Name = "test-comp"
 	comp.Namespace = "default"
 	require.NoError(t, upstream.Create(ctx, comp))
+	comp.ResourceVersion = "1"
```

Comment:
invalidate the copy of the resource received from the server such that the next write will fail until our informers catch up.